src/UI/Pages/Agents/Actions.tsx

```diff
@@ -28,6 +28,7 @@ export const Actions: React.FC<Props> = ({ name, paused }) => {
         isOpen={isOpen}
         isPlain
         onSelect={() => setIsOpen(false)}
+        position="right"
```

Comment:
This should not be necessary.

Reply:
Since it's on the right edge, it is, otherwise it can be partially hidden (maybe it's different on mac?):
![Screenshot from 2022-03-10 11-53-54](https://user-images.githubusercontent.com/55082127/157647501-5f07e7f6-d34a-46fe-81c6-f0c1f8ee7702.png)